RELEASE.md

```diff
@@ -0,0 +1,45 @@
+# Releasing connect-go
+
+This document outlines how to create a release of connect-go.
+
+1. Clone the repo, ensuring you have the latest main.
+
+2. On a new branch, open [connect.go](connect.go) and change the `Version` constant to an appropriate [semantic version](https://semver.org/). To select the correct version, look at the version number of the [latest release](https://github.com/connectrpc/connect-go/releases/latest) and the changes that are included in this new release.
+  * If there are only bug fixes and no new features, remove the `-dev` suffix, set MINOR number to be equal to the [latest release](https://github.com/connectrpc/connect-go/releases/latest), and set the PATCH number to be 1 more than the PATCH number of the [latest release](https://github.com/connectrpc/connect-go/releases/latest).
```

Comment:
Probably a good case for using a footnote for the link target, so you don't have to repeat it.

Reply:
I think this is better as a link than a footnote. Footnote usually have a note for explanation but here we just want to link to the release notes. Left as is (@nicksnyder)

Reply:
I didn't mean to make it _not_ a link. I meant to use footnote style to define the link target, so you don't have to repeat it. See how this is done in the README: https://github.com/connectrpc/connect-go/blob/main/README.md?plain=1#L170-L185

Reply:
TIL: didn't know footnotes could be used like links. Thanks

Reply:
fine with me, I don't care